main.go

```diff
@@ -67,15 +70,26 @@ func main() {
 		*flagWaitInterval = 0.1
 	}
 
+	//->
+	//*flagTarget = "192.168.3.100"
+
 	if *flagTarget == "" {
 		log.Fatal("Missing target (-t 192.168.1.7).")
 	}
 
 	iface, err := net.InterfaceByName(*flagInterface)
+	//iface, err := net.InterfaceByName("以太网")
+	//fmt.Println(iface)
 	if err != nil {
 		log.Fatalf("Could not use interface %s: %v", *flagInterface, err)
 	}
 
+	if runtime.GOOS == "windows" {
+		iface.Name = arp.GetRealCardName(*flagTarget)
+		if err := arp.BindArpByMyself(*flagTarget); err != nil {
```

Comment:
What was the original intention of this method? I tested without it and it worked it fine.